automation/taskcluster/lib/tasks.py

```diff
@@ -627,8 +626,10 @@ def _craft_raptor_task(
             name_prefix, '(on 64-bit-device)' if force_run_on_64_bit_device else ''
         )
 
+        default_apk_artifact_location = 'public/build/{}/geckoNightly/target.apk'.format('armeabi-v7a' if abi == 'arm' else 'arm64-v8a')
+
         apk_url = '{}/{}/artifacts/{}'.format(_DEFAULT_TASK_URL, signing_task_id,
-                                                        DEFAULT_APK_ARTIFACT_LOCATION)
+                                              default_apk_artifact_location)
```

Comment:
Nit: naming. Is this overridable now. Is it just the `apk_artifact_location` (i.e. not just the default one).